### ISSUE TITLE
CW notifications taking a loooong break ?

### ISSUE THREAD

**Issue by Muhammad-Salman-01** [codewars/codewars.com#2503]
It seems starting from a week ago the code wars notifications have been a bit of issue.

You some times get notification, sometimes you don't but its there in you profile (Discourse tab) and sometimes you get none (as in neither the notification and  the discourse tab doesn't show you got a message).

Can also confirm I'm not the only one (since I asked another person) and they too seem to be suffering from it.

**Comment by Blind4Basics** [comment]
yes, CW notification is basically "not working" (...as you'd expect ;o )

from what I remember, the current "rules" are something like:
* you get notifications when...:
  * you're the author of the thread
  * that's a comment on one of your solutions
  * a new issue/suggestion is opened on a kata where you did raise something before (that one is pretty weird, yes)

* you _DON'T_ get notifications when...:
  * the answer is posted with a spoiler flag
  * someone answer to any message you posted while you're _not_ the author of the thread (meaning when "your answer is an answer too")

(I'm not sure this is fully documented in an issue yet, actually...)

**Comment by Muhammad-Salman-01** [comment]
@Blind4Basics : 

Shouldn't I get notification when I'm the author of Kata and also have commented on said post ?

**Comment by farhan443** [comment]
@Blind4Basics Even if there's a spoiler flag, he should still get notified. Because, well, he's the author.
